lisa/tools/date.py

```diff
@@ -32,3 +37,25 @@ def set(self, new_date: datetime) -> None:
             expected_exit_code=0,
             expected_exit_code_failure_message="Failed to set date time.",
         )
+
+
+class DateFreeBSD(Date):
+    @property
```

Comment:
It doesn't need to duplicate the same methods. Just write the set method.

Reply:
Removed all the extra methods and just left the set function.